src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/hmpps/Response.kt

```diff
@@ -2,12 +2,13 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps
 
 data class Response<T>(val data: T, val errors: List<UpstreamApiError> = emptyList()) {
   companion object {
-    fun <T> merge(responses: List<Response<List<T>>>): Response<List<T>> =
-      Response(data = responses.flatMap { it.data }, errors = responses.flatMap { it.errors })
+    fun <T> merge(responses: List<Response<List<T>>>): Response<List<T>> = Response(data = responses.flatMap { it.data }, errors = responses.flatMap { it.errors })
```

Comment:
Strange that all other files so far have expanded the code but this one has compressed it.

Reply:
Kolint prefer assignment in one line when number of parameter is small.